CMakeLists.txt

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.11)
```

Comment:
any reason behind this?

Reply:
We have some workarounds for older CMakes, that could be cleaned up along the way, and I want to use newer FindCuda that has better support for cross-compiling.